kpm/lib/kpm/tasks.rb

```diff
@@ -432,6 +432,97 @@ def system
+        method_option :import,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'import account for a previously exported file.'
+        method_option :tenant_record_id,
+                      :type    => :numeric,
+                      :default => nil,
+                      :desc    => 'replace the tenant_record_id before importing data.'
+        method_option :account_record_id,
+                      :type    => :numeric,
+                      :default => nil,
+                      :desc    => 'replace the account record_id before importing data.'
+        method_option :skip_payment_methods,
+                      :type    => :boolean,
+                      :default => false,
+                      :desc    => 'Skip or swap payment types other than __EXTERNAL_PAYMENT__.'
+        method_option :config_file,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'Yml that contains killbill api connection and DB connection'
+        method_option :killbill_api_credentials,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'Killbill api credentials <api_key> <api_secrets>'
+        method_option :killbill_credentials,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'Killbill credentials <user> <password>'
+        method_option :killbill_url,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'Killbill URL ex. http://127.0.0.1:8080'
+        method_option :database,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'DB connection parameters <database> <user> <password>'
+        desc 'account', 'export/import accounts'
+        def account
+          puts "Please wait processing the request!!!\n"
+          begin
+            config_file = nil
+            if options[:killbill_url] && /https?:\/\/[\S]+/.match(options[:killbill_url]).nil?
+              raise Interrupt,"\e[91;1m--killbill_url, required format -> http(s)://something\e[0m\n\n"
+            end
+
+            if options[:killbill_api_credentials] && options[:killbill_api_credentials].size != 2
+              raise Interrupt,"\e[91;1m--killbill_api_credentials, required format -> <api_key> <api_secrets>\e[0m\n\n"
+            end
+
+            if options[:killbill_credentials] && options[:killbill_credentials].size != 2
+              raise Interrupt,"\e[91;1m--killbill_credentials, required format -> <user> <password>\e[0m\n\n"
+            end
+
+            if options[:database] && options[:database].size != 3
+              raise Interrupt,"\e[91;1m--database, required format -> <database> <user> <password>\e[0m\n\n"
```

Comment:
IMHO, the format is a bit weird / unexpected:

```
kpm > ./bin/kpm account --import kbdump --database 'killbill killbill killbill'
Please wait processing the request!!!
--database, required format -> <database> <user> <password>
kpm > ./bin/kpm account --import kbdump --database killbill killbill killbill
Please wait processing the request!!!
Generating statements
```

I would implement individual switches instead (this would allow us to override only the database name for instance and keep the defaults otherwise). Additionally, support for `ENV` variables would be nice.